src/capsule/CapsuleWallet.ts

```diff
@@ -6,10 +6,43 @@ import * as ethUtil from 'ethereumjs-util'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import { CapsuleSigner } from 'src/capsule/CapsuleSigner'
 import Logger from 'src/utils/Logger'
+import { ChallengeStorageDefault } from './ChallengeStorage'
+import userManagementClient from './UserManagementClient'
 
 const TAG = 'geth/CapsuleWallet'
 
 export class CapsuleWallet extends RemoteWallet<CapsuleSigner> implements UnlockableWallet {
+  // TODO remove me
+  private userId = 'c67b0766-f339-4d86-9c82-fe2410b28460'
+  private biometricStorage = new ChallengeStorageDefault(this.userId)
```

Comment:
I expect we should move it to `CapsuleReactNativeWallet` implementation after merging #6